File: 2619-categorize-box-according-to-criteria/2619-categorize-box-according-to-criteria.cpp
class Solution {
public:
    string categorizeBox(int length, int width, int height, int mass) {
        if(length<=100000&&width<=100000&&height<=100000&&mass<=1000){
        long long volume=static_cast<long long>(length)*width*height;
        string r1="",r2="";

        if(length>=10000||width>=10000||height>=10000||volume>=1000000000){
          r1="Bulky";
          
        }
        if(mass>=100){
            r2="Heavy";}
        if(r1=="Bulky" && r2=="Heavy"){
            return "Both";
        }
        else if(r1!="Bulky" && r2=="Heavy"){
            return "Heavy";
        }
        else if(r1=="Bulky"&& r2!="Heavy"){
            return "Bulky";
        }
        else{
            return "Neither";
        }}
        return  "Neither";
        
    }
};

File: 2619-categorize-box-according-to-criteria/2619-categorize-box-according-to-criteria.py
class Solution:
    def categorizeBox(self, length: int, width: int, height: int, mass: int) -> str:
        if(length<=10**5 and width<=10**5 and height<=10**5 and mass<=10**3):
            r=""
            s=""

            if(length>=10**4 or width>=10**4 or height>=10**4 or length*width*height>=10**9):
                r="Bulky"
            if(mass>=100):
                s="Heavy"
            if(r=="Bulky" and s=="Heavy"):
                return "Both"
            elif(r!="Bulky" and s=="Heavy"):
                return "Heavy"
            elif(r=="Bulky" and s!="Heavy"):
                return "Bulky"
            else:
                return "Neither"

